tmt/convert.py

```diff
@@ -709,6 +718,93 @@ def read_nitrate(
     return common_data, individual_data
 
 
+def read_polarion(
+        data: NitrateDataType,
+        polarion_case_id: Optional[str],
+        link_polarion: bool) -> None:
+    """ Read data from Polarion """
+    # Find Polarion case
+    echo(style('Polarion ', fg='blue'), nl=False)
+    polarion_case = tmt.export.get_polarion_case(data, polarion_case_id=polarion_case_id)
+    if not polarion_case:
+        raise ConvertError('Failed to find test case in Polarion')
```

Comment:
```suggestion
        raise ConvertError('Failed to find test case in Polarion.')
```

See also the [hint](https://tmt.readthedocs.io/en/latest/contribute.html#introduction) for error messages.